src/app/shared/form/button/button.component.scss

```diff
@@ -0,0 +1,15 @@
+button {
+    border: none;
+    color: white;
```

Comment:
colors should be stored in a variable and imported from separated file, i.e. `variables.scss`